ingest/src/ingest/spotify/collector.py

```
@@ -0,0 +1,241 @@
+        # OAuthマネージャーの設定
+        self.auth_manager = SpotifyOAuth(
+            client_id=client_id,
+            client_secret=client_secret,
+            redirect_uri=redirect_uri,
+            scope=scope,
+            open_browser=False,  # ヘッドレス環境ではブラウザを開かない
+        )
+
+        # アクセストークンのリフレッシュ
+        try:
+            token_info = self.auth_manager.refresh_access_token(refresh_token)
```

Ruff (0.14.8)

Local variable `token_info` is assigned to but never used

Remove assignment to unused variable `token_info`

(F841)